site/en/tutorials/keras/save_and_load.ipynb

```diff
@@ -341,9 +341,9 @@
         "id": "wlRN_f56Pqa9"
       },
       "source": [
-        "Create a new, untrained model. When restoring a model from weights-only, you must have a model with the same architecture as the original model. Since it's the same model architecture, you can share weights despite that it's a different *instance* of the model.\n",
```

Comment:
I think "Create a new, untrained model." should be/have been a subtitle, such as `### Create a new, untrained model`. Maybe this paragraph can benefit from minor refactoring/rephrasing (e.g. "despite that" -> "even though", but, overall, it sounds great, I think.

Reply:
The reason In thought it was worth changing was because when I read the original paragraph I felt like I had to untangle it a bit. I think that moving the 'main point' ("As long as two models share the same architecture you can share weights between them.") of the paragraph to the start to provide context to the rest makes it easier to parse.

If you still prefer the original, I'm happy to change it back (and add the subtitle).

Reply:
No worries. By the way, I'm just a passionate community member like yourself :) Thanks for your help!